libertree-frontend-ramaze/view/docs/en_GB/custom-css-file.markdown

```diff
@@ -0,0 +1,55 @@
+# Custom-CSS-File
+You can copy paste the code below in to your settings. Please put behind the semicolons a number in hexadecimal for the color. 
+
+## Clarification
```

Comment:
Maybe this heading is not needed.  The previous section is just two short sentences, and this one is just one sentence long.  Could you try to merge them?